### PR TITLE
Address issues in version dropdown.

### PR DESCRIPTION
**Pull request by etefera**
Addresses #1662 for the new site.
Preview at https://kpt-dev-ee2a9--version-dropdown-98kk6v2g.web.app

### CODE REVIEW

File: site/static/js/plugins.js [hunk 1]
```diff
@@ -43,6 +43,23 @@ function convertFromHugo(content) {
...
 }
 
+async function addVersionDropdown() {
+  const sidebar = document.getElementsByClassName("sidebar").item(0);
+  const versionDropdown = `
+  <div class="dropdown">
+                <button class="btn btn-primary dropdown-toggle" type="button" data-toggle="dropdown">v1.0.0-alpha.3
+                <span class="caret"></span></button>
+                <ol class="dropdown-menu">
+                <li><a href="/installation/">v1.0.0-alpha.3</a></li>
```
Comment (mikebz) [inline site/static/js/plugins.js:53]: this is OK for now but I think the full implementation of a version drop down would preserve the current page and maybe just modify the URL or something along those lines.

**Review by mikebz — APPROVED** [review]
approving as a temporary solution.

File: site/static/js/plugins.js (hunk 1 above)
Comment (mikebz) [reply site/static/js/plugins.js:53]: I would also maybe name it v1 beta1 or something more durable.

Comment (etefera) [reply site/static/js/plugins.js:53]: Will do for now
We should decide on the branch/tag naming scheme for multiple versions of the documentation and tool we want to show publicly. That'll drive the dropdown entries in the future

**Review by frankfarzan — APPROVED** [review]

File: site/static/js/plugins.js (hunk 1 above)
Comment (frankfarzan) [reply site/static/js/plugins.js]: I missed this. What's `v1beta1`? Please use the release semver.

**Pull request merged**